### PR TITLE
Dev: button zoom to extent

### PR DESCRIPTION
**Pull request by EnBakir**
Button that allows a zoom to extent.

### CODE REVIEW

**Comment by taulinger** [comment]
Store/Map infrastructure is set. Next we have to do:
- (more) tests for `ExtentButton.js`
- beautify the layout of the button
- get the default extent from a central service (me)

**Comment by EnBakir** [comment]
Great, then i can continue :+1:

**Comment by taulinger** [comment]
MapService is there...

To get the default extent, you have to `inject('MapService')`, then call `mapService.getDefaultMapExtent()`

File: src/modules/map/i18n/map.provider.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ export const provide = (lang) => {
 				map_info_button: 'Information',
 				map_info_button_help: 'Help',
 				map_info_button_contact: 'Contact',
-				map_info_button_about: 'About us'
+				map_info_button_about: 'About us',
+				map_zoom_extent_button: 'Show whole map'
```
Comment (thiloSchlemmer) [inline src/modules/map/i18n/map.provider.js:13]: I would prefer: 'Zoom to full extent'

File: src/modules/map/i18n/map.provider.js [hunk 2]
```diff
@@ -20,7 +21,8 @@ export const provide = (lang) => {
 				map_info_button: 'Information',
 				map_info_button_help: 'Hilfe',
 				map_info_button_contact: 'Kontakt',
-				map_info_button_about: 'Impressum'
+				map_info_button_about: 'Impressum',
+				map_zoom_extent_button: 'Ganz Bayern anzeigen'
```
Comment (thiloSchlemmer) [inline src/modules/map/i18n/map.provider.js:25]: we can avoid a country-specific text?

**Review by thiloSchlemmer — CHANGES_REQUESTED** [review]
Some hints/questions inline

File: src/modules/map/i18n/map.provider.js (hunk 2 above)
Comment (taulinger) [reply src/modules/map/i18n/map.provider.js:25]: I think, it's OK when we use BayernAtlas specific translations in German

File: src/modules/map/i18n/map.provider.js (hunk 1 above)
Comment (EnBakir) [reply src/modules/map/i18n/map.provider.js:13]: Thanks, that's a good idea !

Comment (taulinger) [reply src/modules/map/i18n/map.provider.js:13]: @EnBakir Can you change that asap, please? We need the infrastructure code...😉

**Review by taulinger — APPROVED** [review]
Go for a merge

**Review by thiloSchlemmer — APPROVED** [review]

**Pull request merged**